pkg/pipelineascode/match.go

```diff
@@ -229,6 +229,15 @@ func (p *PacRun) getPipelineRunsFromRepo(ctx context.Context, repo *v1alpha1.Rep
 		if matchedPRs, err = matcher.MatchPipelinerunByAnnotation(ctx, p.logger, pipelineRuns, p.run, p.event, p.vcx); err != nil {
 			// Don't fail when you don't have a match between pipeline and annotations
 			p.eventEmitter.EmitMessage(nil, zap.WarnLevel, "RepositoryNoMatch", err.Error())
+			// In a scenario where an external user submits a pull request and the repository owner uses the
```

Comment:
one other thing we need to update the documentation explaining this behaviour and that it only happen on github